app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(comment_params)
+    @comment.user_id = current_user.id
+    @comment.post_id = params[:post_id]
+
+    @comment.update_comments_counter
+    if @comment.save
+      redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+    else
+      render :new
+    end
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
```

Comment:
- [ ] I would recommend making `comment_params` the private method, it makes the application secure by permitting only limited attributes.

- [ ] Also add `new` method for CommentsController. Also add `new` path for comments in `routes.rb`

```ruby
class CommentsController < ApplicationController
 def new
    @comment = Comment.new
  end
  ....
end
```